CardiorespiratoryFitness/CardiorespiratoryFitness/iOS/CRFHeartRateSampleProcessor.swift

```diff
@@ -564,19 +558,20 @@ internal class CRFHeartRateSampleProcessor {
     }
 
     func meanFilterOrder(for samplingRate: Int) -> Int {
-        if (samplingRate <= 32) {
-            return 33
+        if samplingRate <= 15 {
+            return 15
```

Comment:
curious why the other cases return the <= comparison value + 1, but this one just returns the value -- is it an odd/even thing? I.e. return value should always be odd?

Reply:
@itismeghasyam 

I meant to ask about that too, or think about it (smile) but I am not sure. Sorry.